src/streamlit_extras/jupyterlite/__init__.py

```diff
@@ -0,0 +1,22 @@
+from streamlit.components.v1 import iframe
+
+from .. import extra
+
+@extra
+def jupyterlite(height:int,width:int):
```

Comment:
Please add a docstring to this function. Look at other extras for inspiration and layout! Thanks ☺️